### PR TITLE
changed functions names to snakecase

### PR DESCRIPTION
**Pull request by RiddhiBagadiaa**

### CODE REVIEW

File: client/src/featureform/dashboard_metadata.py [hunk 1]
```diff
@@ -96,59 +96,59 @@ def training_set_variant(variantData):
...
+def training_sets(row_data):
+    variant_data = training_set_variant(sql_object.get_variant_resource("training_set_variant", "trainingSetName", row_data['name']))
     return TrainingSetResource( 
-                rowData['type'], #type
-                rowData['defaultVariant'], #defaultvariant
-                rowData['name'], #name
-                variantData[0], #variantsDict
-                variantData[1] #all variants
+                row_data['type'], #type
+                row_data['defaultVariant'], #defaultvariant
```
Comment (sdreyer) [inline client/src/featureform/dashboard_metadata.py:113]: Should make dictionary keys lowercase as well

File: client/src/featureform/dashboard_metadata.py [hunk 2]
```diff
@@ -201,9 +201,9 @@ def entities(rowData):
                 rowData['type'], #type
                 rowData['description'], #description
                 rowData['status'], #status
-                variant_organiser(feature_variant(sqlObject.getVariantResource( "feature_variant", "entity", rowData['name']))[2]), #features
-                variant_organiser(label_variant(sqlObject.getVariantResource( "labels_variant", "entity", rowData['name']))[2]), #labels
-                variant_organiser(training_set_variant(sqlObject.getVariantResource( "training_set_variant", "label", rowData['name']))[2]) #training sets
+                variant_organiser(feature_variant(sql_object.get_variant_resource( "feature_variant", "entity", rowData['name']))[2]), #features
+                variant_organiser(label_variant(sql_object.get_variant_resource( "labels_variant", "entity", rowData['name']))[2]), #labels
+                variant_organiser(training_set_variant(sql_object.get_variant_resource( "training_set_variant", "label", rowData['name']))[2]) #training sets
```
Comment (sdreyer) [inline client/src/featureform/dashboard_metadata.py:206]: rowData->row_data

File: client/src/featureform/resources.py [hunk 3]
```diff
@@ -650,12 +650,12 @@ def _create_training_set_resource(self, db) -> None:
 
     def _check_insert_training_set_resources(self, db) -> None:
         try:
-            db.getNameVariant("labels_variant", "labelName", self.label[0], "variantName", self.label[1])
+            db.get_name_variant("labels_variant", "labelName", self.label[0], "variantName", self.label[1])
```
Comment (sdreyer) [inline client/src/featureform/resources.py:653]: Has this branch been rebased with the magic numbers one?

Comment (RiddhiBagadiaa) [reply client/src/featureform/resources.py:653]: Yes it has

File: client/src/featureform/dashboard_metadata.py (hunk 1 above)
Comment (RiddhiBagadiaa) [reply client/src/featureform/dashboard_metadata.py:113]: Ya sorry, I'll ask Ksshiraja to do the rest, there were too many. I just wanted to make sure I didn't break anything before I hand it over so I open a draft PR

**Pull request closed without merging**